liquidprompt

```diff
@@ -381,10 +381,12 @@ __lp_source_config() {
     __lp_foreground_color "$(( 1 + lp_hostname_hash % 6 ))"
     LP_COLOR_HOST_HASH="${_LP_OPEN_ESC}${af_color}${_LP_CLOSE_ESC}"
 
-
     # Default values (globals)
     LP_TIME_FORMAT=${LP_TIME_FORMAT:-"%H:%M:%S"}
     LP_BATTERY_THRESHOLD=${LP_BATTERY_THRESHOLD:-75}
+    LP_DISK_THRESHOLD=${LP_DISK_THRESHOLD:-100000} # 100 Mb in Kb
```

Comment:
`Mb` is short for Megabit, right? So it should be `MB`, or actually `MiB`. And the math that `__lp_bytes_to_human()` uses is based off of 1024 (2^10), not 1000. So this should be `102400`.